manifests/configmap.yaml

```diff
@@ -24,7 +24,7 @@ data:
   # enable_crd_validation: "true"
   # enable_database_access: "true"
   # enable_init_containers: "true"
-  # enable_lazy_image_upgrade: "true"
+  # enable_lazy_spilo_upgrade: "true"
```

Comment:
when the internal default is `false`, it shouldn't be true here. Same for CRD-based configuration.